dotnet/src/Functions/Functions.OpenApi/OpenApiKernelPluginFactory.cs

```diff
@@ -266,6 +266,7 @@ async Task<RestApiOperationResponse> ExecuteAsync(Kernel kernel, KernelFunction
         var additionalMetadata = new Dictionary<string, object?>
         {
             { OpenApiKernelPluginFactory.OperationExtensionsMethodKey, operation.Method.ToString().ToUpperInvariant() },
+            { OpenApiKernelPluginFactory.OperationExtensionsPathKey, operation.Path },
```

Comment:
Should we consider already passing everything in `operations` thats not yet available as information in the metadata? 

i.e `Id`, etc ?

```suggestion
            { OpenApiKernelPluginFactory.OperationExtensionsPathKey, operation.Path },
            { OpenApiKernelPluginFactory.OperationExtensionsIdKey, operation.Id },
            ...
```

Reply:
@SergeyMenshykh what do you think? We could just add the `RestApiOperation` into the metadata.

Reply:
@markwallace-microsoft, I think we should do it but after reviewing the class and its dependencies public API surface.